docs/pages/faq/index.md

```diff
@@ -72,21 +72,27 @@ So you can create a new file named `.nojekyll` in the `public` folder of the pro
 |   |-- .nojekyll
 ```
 
-## 显示的文章创建/修改时间不是 UTC 时间 {lang="zh-CN"}
+## 显示的文章创建/修改时间不正确 {lang="zh-CN"}
 
-## The displayed article creation/modification time is not UTC time {lang="en"}
+## The displayed article creation/modification time is incorrect {lang="en"}
 
 ::: zh-CN
 根据[这份 `YAML` 规范](https://yaml.org/type/timestamp.html)，符合 `ISO 8601` 标准的时间格式都会被解析为 `Date` 类型，且**不显式标注时区的时间戳都会作 UTC 处理**。
 
-但是为了方便写作与从其他框架迁移，我们将未显式标注时区的时间戳解析为 `site.config.ts` 中 `timezone` 对应的时间（即 `2024-07-06 12:00:00` 在 `Asia/Shanghai` 下会解析 为 `2024-07-06T12:00:00+08:00`）。
+但是为了方便写作与从其他框架迁移，我们将未显式标注时区的时间戳解析为**系统时区**对应的时间（即 `2024-07-06 12:00:00` 在 `Asia/Shanghai` 下会解析 为 `2024-07-06T12:00:00+08:00`）。
 
 无论如何，我们建议**显式**添加时区信息，例如：
```

Comment:
In fact, I'm a little confused that explicit additions have migration costs for existing articles. Can't we set this for the user by default by setting the time zone?

Reply:
Using `Date` interpretation of  `node.js` is the best we can do (semantically correct, acts intuitively), which is only determined by process timezone (`process.env.TZ`).

I consider letting user to add this extra step instead of messing up with the nodejs process environment more acceptable.

In worst case scenario, we are just getting the *same behavior* as before (timestamp without timezone information gets coerced to UTC as the host machine is in UTC).

Reply:
I think it's necessary to add more instructions in the documentation, as well as how to set timestamps by various ways (etc node variables...).